tsconfig.json

```
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "declaration": true,
+    "noImplicitAny": false,
+    "noUnusedLocals": false,
+    "removeComments": false,
+    "noLib": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "target": "es6",
+    "sourceMap": false,
+    "outDir": "./dist"
+  },
+  "include": ["src/**/*","tests/**/*", "index.ts"],
```

tests don't need to be compiled and tsc will walk the program from the root so no need to target everything
```suggestion
  "include": ["src/index.ts"],
```